src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -93,11 +101,25 @@ internal abstract partial class DefinitionItem
                 nameDisplayParts, displayIfNoReferences);
         }
 
+        // Kept around for binary compat with F#/TypeScript.
```

Comment:
I would consider that (a source break) acceptable based on their use of internal APIs. I see the importance of preserving the method though for binary compatibility.